### ISSUE TITLE
Missing import in bench module

### ISSUE THREAD

**Issue by mehdisadeghi** [simphony/simphony-common#81]
As I found there has been a refactoring process in place to rename `cuds_file.py` to `h5_cuds_file.py` a month a go. Apparently [the imports](https://exchange.iwm.fraunhofer.de/owa/redir.aspx?C=0531a39da4f541618c93f7a7a7d576cb&URL=https%3a%2f%2fgithub.com%2fsimphony%2fsimphony-common%2fcommit%2f35e785d9936bc8d121037a23d24595dc6a1fabb2) in bench modules are missing this change. The incorrect lines are: 

```
from simphony.io.cuds_file import CudsFile
from simphony.cuds.particle import Particle
```

Moreover these are not covered in tests so nobody has realized this.

**Comment by itziakos** [comment]
Thanks @mehdisadeghi  

It is uncommon for benchmarking code to be part of the travis-ci test suite and run on every commit, the main problems are:
- Benchmarks (in general) take time to run and thus we could run out of the current travis-ci limit
- CI  testing is really effective when tests finish without a few minutes. When that is not true it is difficult to persuade people to do changes on a review when they have to wait 40 minutes to get an answer (e.g the tests in simphony-framework is such a test case).
- Running the benchmarks on travis-ci does not give any useful information since the performance data on a travis-ci VM depend on more variables that just our code.

However, none of the above is a justification for having bench mark code and examples that are broken. In similar situations on production code we usually do the following:

- Add a script that will try to import the example and benchmark code as a module. This is not expensive time-wise (so we can add it in the travis-ci run) and will usually catch import errors.
- Add one more item to manually check when cutting a new release versions. There is probably a script that will gather and run the examples and benchmarks one after the other. The developer can evaluate if errors appear and call for bug fixes.
- Make everybody aware of the example and bench code so that we can pick-up problems during the PR review process.

How about implementing the same workflow here?

**Comment by mehdisadeghi** [comment]
@itziakos I think at least catching import errors is a good idea, then it would not confuse developers specially new ones like myself :).

**Comment by itziakos** [comment]
I agree

**Comment by itziakos** [comment]
this has been addressed in #105
